lib/system.nim

```diff
@@ -57,17 +57,34 @@ include "system/basic_types"
 
 proc compileOption*(option: string): bool {.
   magic: "CompileOption", noSideEffect.}
-  ## Can be used to determine an `on|off` compile-time option. Example:
+  ## Can be used to determine an `on|off` compile-time option.
+  ##
+  ## See also:
+  ## * `compileOption <#compileOption,string,string>`_ for enum options
+  ## * `declared <#declared,untyped>`_
+  ## * `defined <#defined,untyped>`_
+  ## * `define pragmas <manual.html#implementation-specific-pragmas-compileminustime-define-pragmas>`_
+  ##
```

Comment:
also like to `std/compilesettings` which gives a more comprehensive API
(and backlink compileOption from there too)